_posts/2022-07-15-documentation-framework.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: Documentation Framework
+author: Julia Signell
```

Comment:
```suggestion
author: Julia Signell and Jacob Tomlinson
```

Do you mind if I put my name on this too?

Reply:
oh absolutely!